### PR TITLE
Feature/enable basic auth support

### PR DESCRIPTION
**Pull request by axdotl**
Enable basic auth for artifactory

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,20 +43,27 @@ Configure plugin:
...
+        <uploadRepoStable>
+            <name>stable-repo</name>
+            <url>${repoBaseUrl}/helm-stable</url>
+            <!-- Basic authentication is supported from HELM version >= 2.9 -->
+            <username>foo</username>
+            <password>bar</password>
+        </uploadRepoStable>
+        <uploadRepoSnapshot>
+            <name>snapshot-repo</name>
+            <url>${repoBaseUrl}/helm-snapshots</url>
```
Comment (stefanwiedemann) [inline README.md]: Is there no need for authentication on the snapshot repo?

Comment (axdotl) [reply README.md]: This is just an example where I'd like to show that auth is not mandatory.

File: README.md [hunk 2]
```diff
@@ -43,20 +43,27 @@ Configure plugin:
       <configuration>
         <chartDirectory>${project.basedir}</chartDirectory>
         <chartVersion>${project.version}</chartVersion>
-        <helmRepoUrl>${helm.repo.url}</helmRepoUrl>
-        <helmUploadUrlStable>${helm.repo.url}/stable/api/charts</helmUploadUrlStable>
-        <helmUploadUrlSnapshot>${helm.repo.url}/snapshot/api/charts</helmUploadUrlSnapshot>
+        <uploadRepoStable>
+            <name>stable-repo</name>
```
Comment (stefanwiedemann) [inline README.md]: Why did you decide to use a baseUrl with fixed paths for stable and snapshot, instead of configuring both addresses?

Comment (stefanwiedemann) [reply README.md]: I see it. Missread the diff...

**Pull request merged**